packages/minuteman/build

```diff
@@ -19,6 +19,7 @@ Description=Layer 4 Load Balancer: DC/OS Layer 4 Load Balancing Service
 Restart=always
 StartLimitInterval=0
 RestartSec=5
+LimitNOFILE=16384
```

Comment:
maybe also explain in the pr description why we are doing this since it's unrelated to the parser